services/post-service/AI_ENDPOINTS.md

```
@@ -0,0 +1,107 @@
+## Overview
+The post-service now includes AI-powered content and caption generation using Google's Gemini model through LangChain.
+
+## Setup
+
+### Prerequisites
+1. Google API Key for Gemini AI
+2. LangChain dependencies installed
+
+### Environment Variables
+Add the following to your `.env` file:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)